pkg/migrations/mls/20240411200242_init-identity.up.sql

```diff
@@ -0,0 +1,34 @@
+SET statement_timeout = 0;
+
+--bun:split
+
+CREATE TABLE inbox_log (
+    sequence_id BIGSERIAL PRIMARY KEY,
+    inbox_id TEXT NOT NULL,
+    commit_status SMALLINT NOT NULL,
+    server_timestamp_ns BIGINT NOT NULL,
+    identity_update_proto BYTEA NOT NULL
+);
+
+--bun:split
+
+CREATE INDEX idx_inbox_log_inbox_id_commit_status ON inbox_log(inbox_id, commit_status);
+
+--bun:split
+
+CREATE TABLE address_log (
+    inbox_log_sequence_id BIGINT PRIMARY KEY,
```

Comment:
In the notes above it says 

> the address_lookup_cache stores the inbox_log_sequence_id

But you have that field only on the `address_log` here.